userdocs/theme/partials/header.html

```diff
@@ -75,5 +75,14 @@
       }
       toggleSwitch.addEventListener('click', switchTheme);
     </script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-GBCDQ2QFTZ"></script>
+    <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'G-GBCDQ2QFTZ');
```

Comment:
Minor alignment nit:

```suggestion
        window.dataLayer = window.dataLayer || [];
        function gtag(){dataLayer.push(arguments);}
        gtag('js', new Date());
    
        gtag('config', 'G-GBCDQ2QFTZ');
```